t/react_template/README.md

```diff
@@ -0,0 +1,92 @@
+**This is not an official Google product.**
+
+# Template: React Framework
+
+This template was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). 
+
+The react template enpower the front end development flow by adding automate Continuous Build (CB) and Continuous Testing (CT). GitHub Actions are leveraged to perform a series of pre-submit validations. 
+
+A branch protection rule will be established to enforce checks on each pull request including
+
+- Lint to provide static analysis (with ESlint)
+- Code format (with Prettier)
+- Build (with React-scripts)
+- Unit test and Component test
+- Integration test (with Cypress)
+
+
+## Set up instruction
+
+1. cd into an empty directory
+
+    ```shell
+    $ mkdir ~/react_template
+    $ cd ~/react_template
+    ```
+1. Install the `abc` binary
+
+    ```shell
+    $ go install github.com/abcxyz/abc/cmd/abc@latest
+    $ abc --help
+    ```
+
+    This only works if you have go installed (https://go.dev/doc/install) and have the Go binary directory in your $PATH (try PATH=$PATH:~/go/bin).
+
+
+1. Execute the template defined in the `t` directory.
+This will output a file named `main.go` in your working directory containing
+the transformed program.
+
+    ```shell
+    $ abc templates render github.com/abcxyz/abc.git//t/react_template
+    ```
+
+## Available Scripts
+prerequests: `node.js` and `npm`. If not, follow the download steps [here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
```

Comment:
Forgive my ignorance, but what's the purpose of these backslashes?

Reply:
We might not need that. For other read me we simply tolerate long sentences. E.g. line 86

Reply:
removed, Done